src/styles/game/panes/index.css

```
@@ -1,5 +1,6 @@
 @import url('./game.css');
 @import url('./settings.css');
+@import url('./theme-selection.css');
```

Stylelint (17.14.0)

[error] Expected "url('./theme-selection.css')" to be "'./theme-selection.css'" (import-notation)

(import-notation)